docs/translating/polish.md

```diff
@@ -81,3 +83,43 @@ Zulip friendly and usable.
 **view**: widok
 
 **starred message**: oznaczona wiadomość
+
+**message**: wiadomość
+
+**stream**: kanał
+
+**public stream**: kanał publiczny
+
+**bot**: bot
+
+**Integration**: integracja
+
+**Notification**: Powiadomienie
+
+**alert word**: ostrzeżenie
+
+**narrow to**: zwięź do
+
+**mute**: wycisz
+
+**unmute**: wyłącz wyciszenie
+
+**deactivate**: dezaktywować
+
+**reactivate**: reaktywować
+
+**search**: znajdź
+
+**pin**: przypnij
+
+**invalid**: niepoprawny/a
+
+**i want**: chcę
+
+**user**: użytkownik
+
+**person**: osoba
+
+**people**: ludzie
```

Comment:
I'm not a native speaker, so I cannot comment on the correctness of the translations here. In any case, you should add context for at least some of these translations, providing examples of how they can be used, what to watch out for, etc.